include/dpp/message.h

```diff
@@ -1523,6 +1540,13 @@ struct DPP_EXPORT message : public managed {
 	 * @return true if message is a DM
 	 */
 	bool is_dm() const;
+
+	/**
+	 * @brief Returns true if any of attachments is remixed
+	 * 
+	 * @return true if any of attachments is remixed
+	 */
+	bool is_remix() const;
```

Comment:
i dont see much point in adding this. Just having is_mixed() to check for attachments and nothing more seems pointless. It's a copy and paste from attachment and better placed only in dpp::attachment. Also the name of this method implies that "the message" "is remixed". If we're adding this, at least rename it to something like `has_remix_attachment`.